django/conf/urls/i18n.py

```diff
@@ -4,7 +4,7 @@
 from django.views.i18n import set_language
 
 
-def i18n_patterns(*urls):
+def i18n_patterns(*urls, prefix_default_language=True):
```

Comment:
Unfortunately this is not a valid syntax on Python 2. Use `**kwargs` and `get('prefix_default_language', True)` from it.